app/views/dashboard/_subscription_widget.html.erb

```diff
@@ -4,11 +4,6 @@
 
 <% if Organization.current.blank? %>
   <p class="ca"><%= _("Please select an organization to view subscription status.") %></p>
-<% elsif Organization.current.simple_content_access?%>
-  <div class="bastion alert alert-info">
-    <span translate>This organization has Simple Content Access enabled. Hosts can consume from all repositories in their Content View regardless of subscription status.
-    </span>
-  </div>
```

Comment:
I realized that since we removed the Subscription Status widget if SCA is enabled, this section isn't needed at all.